### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.2.1 (2020-04-23)
+
+- Fixed the label component to have space between the label icon and text. [#77](https://github.com/blackbaud/skyux-indicators/pull/77)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: I'd swap out "have" for "add."

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.1](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.1)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.1](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.1)

**Pull request merged**